README.md

```diff
@@ -46,6 +46,38 @@ The site's default CSS has now moved to a new place within the gem itself, [`ass
 
 --
 
+When you override only a minima-sass-partial, it is not automatically imported because we're still importing the `minima.scss` within the theme-gem and that subsequently imports the partials with respect to itself, i.e. partials within the gem. Hence you should either include a *copy of `minima.scss` from the gem* inside the `_sass` directory at source or the overriding `/assets/main.scss` file should explicitly import the edited partial. :
+  e.g. To have an **edited** `_syntax-highlighting.scss` be rendered, you should either have
+
+```sass
+/* <your-site>/assets/main.scss */
+
+@import "minima";
+@import "minima/syntax-highlighting";
```

Comment:
Perhaps I wasn't clear enough, but if you read again, you'll see that the two contexts are different.
The first scenario is **`importing`** the partials to the SCSS file in `assets/` while the second scenario is having a copy of the `_sass/` directory with necessary partials.

The common factor in both scenarios or, the takeaway from this entire section is that:

> ***To override `minima/syntax-highlighting.scss` partial, one will need to pull `minima.scss` from their `<source>` either by importing it again, or by having it physically present.***